package.json

```diff
@@ -76,9 +76,9 @@
     "style-loader": "^0.19.0",
     "stylus": "^0.54.5",
     "stylus-loader": "^3.0.1",
-    "webpack": "^4.8.3",
-    "webpack-cli": "^2.1.3",
-    "webpack-dev-server": "^3.1.4"
+    "webpack": "^4.45.0",
+    "webpack-cli": "^3.1.1",
+    "webpack-dev-server": "^3.11.1"
```

Comment:
1. Does it mean that `node 15` + `npm 8` is now supported?
2. How hard it would be to upgrade webpack to 5.x?

Reply:
1. I believe you meant `npm 7` not `npm 8`. If so then yes, it works with `node 15 + npm 7` now
2. This question needs further investigation